### ISSUE TITLE
why we need to concat syn and ugc together

### ISSUE THREAD

**Issue by chenyu1014** [pavancm/CONTRIQUE#23]
Please tell me  why we need to concat syn and ugc together
        syn_i1 = syn_i1.cuda(non_blocking=True)
        ugc_i1 = ugc_i1.cuda(non_blocking=True)
        x_i1 = torch.cat((syn_i1,ugc_i1),dim=0)
        
        #image 2
        syn_i2 = syn_i2.cuda(non_blocking=True)
        ugc_i2 = ugc_i2.cuda(non_blocking=True)
        x_i2 = torch.cat((syn_i2,ugc_i2),dim=0)

**Comment by pavancm** [comment]
This is done only during training. This is done to ensure that the model learns equally from both syn and ugc images. We observed that using only one of them was making the model biased towards those kind of distortions, so we concatenated syn and ugc features to alleviate this issue

**Comment by chenyu1014** [comment]
Thank you！！！
